src/app/ssmt/explore-opportunities/explore-opportunities-form/boiler-form/boiler-form.component.html

```diff
@@ -5,221 +5,271 @@
+        <div class="text-muted text-center baseline-value-input">
+          <span *ngIf="baselineForm.value.preheatMakeupWater == true">Yes</span>
+          <span *ngIf="baselineForm.value.preheatMakeupWater == false">No</span>
         </div>
-        <span class="alert-danger pull-right small"
-          *ngIf="baselineForm.controls.approachTemperature.invalid && !baselineForm.controls.approachTemperature.pristine">
-          <span *ngIf="baselineForm.controls.approachTemperature.errors.min">Value can't be less than
-            {{baselineForm.controls.approachTemperature.errors.min.min}}
-            <span [innerHTML]="settings.steamTemperatureMeasurement | settingsLabel"></span>.</span>
-        </span>
       </div>
     </form>
-    <form [formGroup]="modificationForm" *ngIf="showPreheatBlowdownWater">
-      <div class="form-group">
-        <label for="modificationPreheatBlowdownWater">Modification Preheat Make-up Water</label>
-        <select name="modificationPreheatBlowdownWater" id="modificationPreheatBlowdownWater" class="form-control"
+    <form class="explore-opps-form" [formGroup]="modificationForm">
+      <div class="form-group explore-opps-formgroup">
+        <label for="modificationPreheatBlowdownWater" class="baseline-value-label">Preheat Make-up Water</label>
+        <select name="modificationPreheatBlowdownWater" id="modificationPreheatBlowdownWater" class="form-control w-100"
           (focus)="focusField('preheatMakeupWater')" (change)="save()" formControlName="preheatMakeupWater">
           <option [ngValue]="true">Yes</option>
           <option [ngValue]="false">No</option>
         </select>
       </div>
-      <div class="form-group">
-        <label for="modificationApproachTemperature">Modification Approach Temperature</label>
+    </form>
+    </li>
+
+    <li class="d-flex justify-content-end" [ngClass]="{'justify-content-between': baselineForm.controls.approachTemperature.value}">
+    <form *ngIf="baselineForm.controls.approachTemperature.value" class="explore-opps-form" [formGroup]="baselineForm">
+      <div class="text-center">
+        <label for="baselineApproachTemperature" class="baseline-value-label"> Approach Temperature</label>
+        <div class="text-muted baseline-value-input">{{ baselineForm.controls.approachTemperature.value}} 
+          <span [innerHTML]="settings.steamTemperatureMeasurement | settingsLabel"></span>
+      </div>
+      <span class="alert-danger pull-right small"
+      *ngIf="baselineForm.controls.approachTemperature.invalid && !baselineForm.controls.approachTemperature.pristine">
+      <span *ngIf="baselineForm.controls.approachTemperature.errors.min">Value can't be less than
+        {{baselineForm.controls.approachTemperature.errors.min.min}}
+        <span [innerHTML]="settings.steamTemperatureMeasurement | settingsLabel"></span>.</span>
+    </span>
+      </div>
+    </form>
+    <form class="explore-opps-form" [formGroup]="modificationForm">
```

Comment:
Approach temperature should only appear if "Preheat Make-up Water" option is "Yes". Looks like baseline is working properly but the logic may be incorrect. Baseline display has *ngIf approachTemperature has a value, it should be dependent on the "Preheat Make-up Water" option.